src/client/datascience/interactive-window/nativeInteractiveWindowCommandListener.ts

```diff
@@ -422,15 +430,19 @@ export class NativeInteractiveWindowCommandListener {
         }
     }
 
-    private expandAllCells() {
-        const interactiveWindow = this.interactiveWindowProvider.activeWindow;
+    private expandAllCells(uri?: Uri) {
+        const interactiveWindow = uri
+            ? this.interactiveWindowProvider.windows.find((window) => window.notebookUri!.toString() === uri.toString())
```

Comment:
From the code it looks like notebookUri is always defined, but the "!" feels a tiny bit less typesafe here since it is optional on the interface. Would ? accomplish the same thing since undefined won't equal uri.toString()?

Reply:
Ohh, looks like you have ? below for exportAs (was just reading on more). Can we use that in these locations as well.